backend/market/models.py

```diff
@@ -8,51 +8,81 @@
 
 class Offer(models.Model):
     class Meta:
-        constraints = [models.UniqueConstraint(fields=["user", "sublet"], name="unique_offer")]
+        constraints = [models.UniqueConstraint(fields=["user", "item"], name="unique_offer_market")]
 
-    user = models.ForeignKey(User, on_delete=models.CASCADE, related_name="offers_made")
-    sublet = models.ForeignKey("Sublet", on_delete=models.CASCADE, related_name="offers")
+    user = models.ForeignKey(User, on_delete=models.CASCADE, related_name="offers_made_market")
```

Comment:
are these `_market` because sublet exists? If so let's make a note to fix this after we delete sublet stuff

Reply:
I'll just remove related_name since there's only one ForeignKey to User in Offer.